### DISCUSSION TITLE
arxiv 预印本 bibtex 编译的参考文献 显示不全

### DISCUSSION THREAD

**Discussion by aaronhd** [stone-zeng/fduthesis#292] [Q&A]
使用Google scholar 生成的预印本bibtex  编译后显示不全 文献种类为【A】，缺少其他信息

**Comment by sikouhjw** [comment]
具體點，什麽叫缺少其他信息？

**Reply by aaronhd** [reply]
![搜狗截图20230307211059](https://user-images.githubusercontent.com/9550439/223436175-61d5d877-0232-4bdd-8acf-84e8cd3892ce.png)

如图只能显示到[A].2018   ----> 希望的是[J]//arXiv preprint arXiv:1811.02815. 2018

**Reply by note286** [reply]
`.bib`里怎么写的。

**Reply by aaronhd** [reply]
> `.bib`里怎么写的。

![2222](https://user-images.githubusercontent.com/9550439/223437900-0da2a97f-b9e3-4cef-95e6-b4c2b8642c84.png)

**Reply by note286** [reply]
直接markdown语法贴代码。

**Reply by aaronhd** [reply]
> 直接markdown语法贴代码。

抱歉 ~~~ thesis 整的人有点懵

**Reply by note286** [reply]
> 抱歉 ~~~ thesis 整的人有点懵

你直接给代码，就可以直接复现问题，解决问题，总不能麻烦别人对着图片还原文本吧。

**Reply by sikouhjw** [reply]
https://github.com/zepinglee/gbt7714-bibtex-style/issues/89
